src/TelegramUpdates.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace NotificationChannels\Telegram;
+
+use JsonSerializable;
+use NotificationChannels\Telegram\Traits\HasSharedLogic;
+
+/**
+ * Class TelegramUpdates.
+ */
+class TelegramUpdates implements JsonSerializable
+{
+    use HasSharedLogic;
```

Comment:
I think we can remove this since there are some unnecessary methods from that trait. We can adapt some of those methods later if required.

Reply:
Done.